ts/Shared/BaseForm.ts

```diff
@@ -163,7 +163,7 @@ abstract class BaseForm {
         }
 
         // Add close button.
-        popupDiv.appendChild(popupCloseButton);
+        popupDiv.appendChild(popupCloseButton); // Is this necessary?
```

Comment:
This is the close button you are referring to, and it is needed to be rendered from scratch because all content is cleared when you are showing a new popup.